mitosheet/src/mito/types.tsx

```diff
@@ -717,8 +717,11 @@ export enum MitoEnterpriseConfigKey {
     DISPLAY_SCHEDULING = 'MITO_CONFIG_FEATURE_DISPLAY_SCHEDULING',
     LLM_URL = 'MITO_CONFIG_LLM_URL',
     ANALYTICS_URL = 'MITO_CONFIG_ANALYTICS_URL',
+    JUPYTER_LOG_SERVER_URL = 'MITO_CONFIG_JUPYTER_LOG_SERVER_URL',
+    JUPYTER_LOG_SERVER_BATCH_INTERVAL = 'MITO_CONFIG_JUPYTER_LOG_SERVER_BATCH_INTERVAL',
```

Comment:
As per the spec, we wanted to provide different logging destinations depending on if they are in Streamlit or Jupyter, so that's why I settled on these names.